src/AppBundle/Repository/CommitteeMembershipRepository.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\CommitteeMembership;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\QueryBuilder;
+use Ramsey\Uuid\Uuid;
+
+class CommitteeMembershipRepository extends EntityRepository
+{
+    /**
+     * Returns whether or not the given adherent is already an host of at least
+     * one committee.
+     *
+     * @param string $adherentUuid
+     *
+     * @return bool
+     */
+    public function isCommitteeHost(string $adherentUuid): bool
+    {
+        $adherentUuid = Uuid::fromString($adherentUuid);
+
+        $query = $this
+            ->createQueryBuilder('cm')
+            ->select('COUNT(cm.uuid)')
+            ->where('cm.adherentUuid = :adherent')
+            ->andWhere('cm.privilege = :privilege')
+            ->setParameter('adherent', (string) $adherentUuid)
+            ->setParameter('privilege', CommitteeMembership::COMMITTEE_HOST)
+            ->getQuery()
+        ;
+
+        return (int) $query->getSingleScalarResult() >= 1;
+    }
+
+    /**
+     * Returns whether or not an adherent is already member of a committee.
+     *
+     * @param string $adherentUuid
+     * @param string $committeeUuid
+     *
+     * @return bool
+     */
+    public function isMemberOf(string $adherentUuid, string $committeeUuid): bool
+    {
+        $query = $this
+            ->createMembershipQueryBuilder($adherentUuid, $committeeUuid)
+            ->select('COUNT(cm.uuid)')
+            ->getQuery()
+        ;
+
+        return 1 === (int) $query->getSingleScalarResult();
```

Comment:
Yeah but here we just need to know that it's not empty, so as above it should be the same:
```diff
-return 1 === (int) $query->getSingleScalarResult();
+return (bool) $query->getSingleScalarResult();
```
Am I missing something?